src/components/fields/SchemaField.js

```diff
@@ -156,6 +160,7 @@ function SchemaField(props) {
   return (
     <Wrapper
       label={props.schema.title || schema.title || name}
+      description={props.schema.description || schema.description}
```

Comment:
I don't have a preference, actually. Just kept it consistent with the previous expression that sets the `label` prop.